supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkBackendFactory.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.backend.impl.spark
+
+import akka.actor.{ActorRef, Props}
+import cromwell.backend._
+import cromwell.backend.sfs.SharedFileSystemExpressionFunctions
+import cromwell.backend.io.JobPaths
+import cromwell.core.CallContext
+import wdl4s.Call
+import wdl4s.expression.WdlStandardLibraryFunctions
+
+case class SparkBackendFactory(configurationDescriptor: BackendConfigurationDescriptor) extends BackendLifecycleActorFactory {
```

Comment:
Just a heads up, when you rebase onto the latest develop, this constructor is also passed an extra `actorSystem: ActorSystem` argument. Of course, you can just do nothing with it if you don't need it